docs/exclusive-content.md

```diff
@@ -212,3 +204,41 @@ If you see the source files instead of the example, click **View App** in the bo
     style={{ height: '100%', width: '100%', border: '0' }}>
   </iframe>
 </div>
+
+## Exclusive content with payment verification
+
+The above examples only hide content client side which could be spoofed by a clever user.
+Since the introduction of STREAM receipts it is possible to verify payments using a STREAM receipt verifier.
```

Comment:
```suggestion
Since the introduction of STREAM receipts it is possible to verify payments using a [STREAM receipt verifier](/docs/receipt-verifier).
```